themes/base.theme.bash

```diff
@@ -371,7 +371,9 @@ function chruby_version_prompt {
 }
 
 function ruby_version_prompt {
-  echo -e "$(rbfu_version_prompt)$(rbenv_version_prompt)$(rvm_version_prompt)$(chruby_version_prompt)"
+  if [ -z "${RUBY_VERSION_PROMPT_DISABLED}" ]; then
```

Comment:
I'd prefer a slightly different approach for this. For other, similar cases, we have used the following pattern:

* Define a variable with a default value at the top of the file, e.g. here: https://github.com/Bash-it/bash-it/blob/b35d41464c0bd390e573f7423eaaa63666521c70/themes/base.theme.bash#L71
* The variable name in your case should probably look like this: `THEME_SHOW_RUBY_PROMPT`, default value `true`.
* To check for the variable (here in this function), use code similar to this: https://github.com/Bash-it/bash-it/blob/b35d41464c0bd390e573f7423eaaa63666521c70/themes/base.theme.bash#L433-L437

Reply:
Thanks, nwinkler. as your recommend, the `THEME_SHOW_RUBY_PROMPT` added, default value is true.
if user want hide the ruby version prompt, can simply 
export THEME_SHOW_RUBY_PROMPT='false'
in .bash_profile or any other config file.